uwblueprint/childrens-aid-society
# Create existing provider modal

## Notion ticket link

[Create existing provider modal](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* created a new common modal component for use for future modals
* the isOpen and onClose parameters are passed in from a useDisclosure hook in the parent calling the modal component so that the parent can register when the modal is called based on a button click
* currently an example is in the homepage for the existing provider modal



## Steps to test
1. go to branch
2. go to localhost:3000/ and scroll all the way down
3. at the bottom click the open modal button to see the modal for the existing providers



## What should reviewers focus on?
* if the existing provider modal should be imported somewhere else (i couldn't find the provider screen)
* if the modal looks as expected

<img width="1276" alt="Screen Shot 2022-11-11 at 5 51 35 PM" src="https://user-images.githubusercontent.com/77185514/201441338-9df0b039-cdcd-4246-ad4e-77758f5d2eb2.png">


## Checklist
